src/AppSettings/components/Settings.tsx

```diff
@@ -64,7 +64,7 @@ export const BiometricLockSetting = React.memo(function BiometricLockSetting(pro
   return (
     <AppSettingsItem
       actions={
-        <SettingsToggle checked={props.enrolled && props.value} disabled={!props.enrolled} onChange={props.onToggle} />
+        <SettingsToggle checked={props.enrolled && props.value} disabled={!props.enrolled} onChange={() => undefined} />
```

Comment:
Let's add a comment here why we set a do-nothing click handler here. LGTM afterwards 👍

Reply:
Done